plugin.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#Enter your directory below where plugin is located or place the script in the plugin directory
+PLUGIN_DIR="."
+
+ZIP_FILE="$PLUGIN_DIR/woocommerce-plugin.zip"
+
+EXCLUDE_LIST=(
+    "$PLUGIN_DIR/.git/*"
+    "$PLUGIN_DIR/.plugin.sh"
+)
+
+
+if [ -f "$ZIP_FILE" ]; then
```

Comment:
Won't it be better to add a prompt asking the user whether to overwrite the existing file ?